### PR TITLE
Add multiple queries support in Media Field and Poster Image

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #126

### CODE REVIEW

File: provisioning/dashboards/postgres.json [hunk 1]
```diff
@@ -204,11 +204,11 @@
         "mediaSources": [
           {
             "field": "concat",
-            "id": "9b9286f8-4d82-4131-a8dd-00f5c0a660d6",
+            "id": "image-2-5-2-0-ver",
```
Comment (mikhail-vl) [inline provisioning/dashboards/postgres.json:207]: Why there are hardcoded id in multiple datasources?

File: src/migration.ts [hunk 2]
```diff
@@ -97,7 +102,12 @@ export const getMigratedOptions = (panel: PanelModel<OutdatedPanelOptions>): Pan
      * Migrate imageUrl
      */
     if (options.hasOwnProperty('imageUrl')) {
-      const imageSource = { type: MediaFormat.IMAGE, id: 'image-1-5-2-0-ver', field: options.imageUrl ?? '' };
+      const imageSource = {
+        type: MediaFormat.IMAGE,
+        id: 'image-1-5-2-0-ver',
```
Comment (mikhail-vl) [inline src/migration.ts:107]: Can it be generated uuid?

**Review by mikhail-vl — CHANGES_REQUESTED** [review]
Update hardcoded Ids.

File: src/migration.ts (hunk 2 above)
Comment (vitPinchuk) [reply src/migration.ts]: uuid added to migration flow

File: src/utils.ts [hunk 3]
```diff
@@ -113,7 +113,11 @@ export const getMediaValue = (
 ) => {
   if (series && series.length) {
     for (const item of mediaSources) {
-      const mediaItem = series[0].fields.find((media) => media.name === item.field);
+      const frame = series.find((sItem) =>
```
Comment (asimonok) [inline src/utils.ts:116]: please use findField

File: src/utils.ts [hunk 4]
```diff
@@ -161,3 +165,45 @@ export const getMediaValue = (
...
+ * @param series
+ * @param field
+ */
+export const getValuesForMultiSeries = (series: DataFrame[], fieldName: string) => {
+  const fieldMatcher = fieldMatchers.get(FieldMatcherID.byName);
+  const matcher = fieldMatcher.get(fieldName);
+
+  let currentValues: string[] = [];
+
+  for (const frame of series) {
```
Comment (asimonok) [inline src/utils.ts:200]: please use findFIeld

**Review by asimonok — CHANGES_REQUESTED** [review]

File: src/migration.ts (hunk 2 above)
Comment (vitPinchuk) [reply src/migration.ts]: provisioning updated

File: provisioning/dashboards/postgres.json (hunk 1 above)
Comment (vitPinchuk) [reply provisioning/dashboards/postgres.json]: provisioning updated

File: src/utils.ts (hunk 3 above)
Comment (vitPinchuk) [reply src/utils.ts]: updated

File: src/utils.ts (hunk 4 above)
Comment (vitPinchuk) [reply src/utils.ts]: updated

**Review by asimonok — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["volkovlabs"]
```
[the remark this answers is not in the document]
Comment (vitPinchuk) [reply .vscode/settings.json]: removed

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**